homeassistant/components/media_player/__init__.py

```diff
@@ -543,6 +544,11 @@ def media_duration(self) -> int | None:
         """Duration of current playing media in seconds."""
         return self._attr_media_duration
 
+    @property
+    def media_genre(self) -> str | None:
```

Comment:
It's even strange that "genre" wasn't already in use....
After all the core component of "media_player" shows very few attributes that it would require a complete rewrite to allow it to be modernized a bit. Virtually all media/video/audio meta-tags/metadata are missing and not shown: it doesn't show the Genre, not even the Year of production, we can't put the volume bar directly in the widget to improve Controls...well, it's very-very limited.

However it's not a problem to remove just a new attribute if it creates a lot of inconvenience for you. It's incredible, but ok.

Reply:
> https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model
In your link it is clearly written: "_Only additions will be considered_" (without an architecture discussion).
This is an ADDITION and not a change or a removal. 

It's very annoying waiting days to see as approved a simple addition of an attribute, especalially if it should already be added from the near beginning of media_player (like other lacking attributes) 😅

So what happens now? If you obey me to delete that, I have to remove same attribute also from the original webostv component to not let it assigns a not-more-existent media_player attribute by preventing people to have the possibility to have an additional attribute.

I vote to NOT remove this but I won't insist so much because it seems useless to have to waste so much time for such simple banalities and I prefer to cancel the PR and use it only on my version of repository.

Reply:
This is written in the link:

> Only additions will be considered that are common features among various vendors.

Either remove this change of the entity model or create an architecture discussion and wait for approval of that discussion.